### ISSUE TITLE
add server to test fv_converter

### ISSUE THREAD

**Issue by kmaehashi** [jubatus/jubatus#1105]
Currently jubaconv does not support IDF calculation. It is better to support it.

I expect two IDF calculation modes:

* "online" IDF mode: frequency is incrementally updated when each data is observed
* "batch" IDF mode: frequency is measured from all data before calculating weights

**Comment by kmaehashi** [comment]
I just think it is better to develop a different tool rather than modifying jubaconv. Maybe using RPC interface like in convchecker. http://blog.jubat.us/2012/09/jubaconvchecker.html

**Comment by kmaehashi** [comment]
As discussed in discussed on 2016-05-09, we'll go with the server idea as commented above.

I'm going to write a server that has 2 APIs:

```cpp
/*
 * Update the weight manager using given datum
 * and return weighted feature vector.
 */
map<string,float> update(datum d);

/*
 * Returns weighted feature vector, without
 * updating the weight manager.
 */
map<string,float> calc_weight(datum d);
```

**Comment by kmaehashi** [comment]
Fixed via #1116
